nfr-verified-funding/src/test/java/no/sikt/nva/funding/verified/nfr/handlers/NfrApiStubber.java

```diff
@@ -99,7 +98,7 @@ public String withMatchingLeadNameAsMostRelevantHit(int numberOfMatchesByName, i
         var responseObject = new NfrFundingSearchResult(totalHits, from, size, matches);
 
         var url = "/search?query=" +  URLEncoder.encode(leadName, StandardCharsets.UTF_8)
-                                         .replace("+", "%2B")
+                                         .replace("+", "%20")
```

Comment:
I'm not happy about this, URLEncoder::encode changes spaces to "+", here, which then is replaced for some reason, however, we want to encode spaces as spaces directly. I don't understand why this worked previously if I'm honest.

Reply:
`.replace("+", "%2B")` is replacing  "+" with the encoded form of "+". As far as I can figure out, it shouldn't matter if a real space is encoded as `+` or `%20` anyway. It might matter if a real plus symbol is encoded as a space though:

> "Within the query string, the plus sign is reserved as shorthand notation for a space. Therefore, real plus signs must be encoded." (https://www.w3.org/Addressing/URL/uri-spec.html)

In fact, all three `.replace` steps here seem to be repeating the exact same encoding that `URLEncoder.encode` has already done. This is a bit odd.

Reply:
I remember we had something similar in cristin proxy. Was removed with new version of nva commons